fml/status.h

```diff
@@ -0,0 +1,91 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_FML_STATUS_H_
+#define FLUTTER_FML_STATUS_H_
+
+#include "flutter/fml/string_view.h"
+
+namespace fml {
+
+/// Google's canonical list of errors, should be in sync with:
+/// https://github.com/googleapis/googleapis/blob/master/google/rpc/code.proto
```

Comment:
I don't think there should be any expectation that these should be in sync with grpc. We don't want folks converting the raw code from the status objects we use in the engine and send them over the wire.

Reply:
Done